test/src/org/labkey/test/tests/panoramapremium/TargetedMSQCPremiumTest.java

```diff
@@ -129,7 +130,7 @@ public void testNotification()
         assertEquals("Mismatch in the expected number of notification", 1, notifications.getEmailCount());
         EmailRecordTable.EmailMessage message = notifications.getMessageWithSubjectContaining("Panorama QC Notification");
         notifications.clickMessage(message);
-        assertTextPresent("38 outliers"); //Total outliers
+        assertTextPresent("0 outliers"); //Total outliers
```

Comment:
This seems odd to me, but I guess maybe all the outliers in the previous version were CUSUM and Moving Range?

Reply:
Yes, that is correct. I can probably add a comment here.

Reply:
That's probably not needed. What caught my eye was the dramatic drop from this change. We don't have comments in other places explaining the outlier counts.